### ISSUE TITLE
FR: Search, open & close tab groups

### ISSUE THREAD

**Issue by kvsidharth** [babyman/quick-tabs-chrome-extension#346]
Since chrome supports tab groups now, it would be great if I can search for a tab group in quick tabs. Also, we should be able to open/close the whole tab group from quick tabs.

**Comment by babyman** [comment]
I agree, I think that makes a ton of sense. I have to find some time to check out the API and see what's possible. 

Evan

**Comment by timkgh** [comment]
+1 for searching by tab group name by default and via a `/` command.

**Comment by rbroggi** [comment]
Huge +1, I was starting to build my own plugin for that but realized that `quick-tabs` could be the perfect place for me to spend my cognitive load. As much as I'm a backend engineer (therefore not js expert) I would love to contribute to this feature if you are willing to have some help.

**Comment by rbroggi** [comment]
@babyman if you give me some general directions I can try to work to implement this feature.
